gulpfile.js

```diff
@@ -296,7 +254,7 @@ gulp.task('watch',
 
 // iconfont, less, kss_bootrap_src
 
-gulp.task('dist', [ 'iconfont', 'less', 'styleguide', 'deploy']);
```

Comment:
The icon fontfiles are always getting recreated and comited even if the font isn't changing